app/views/agent/SignUpComplete.scala.html

```diff
@@ -71,7 +71,7 @@ <h2 class="govuk-heading-m">@messages("agent.sign-up-complete.whatNow.heading")<
         <p class = "govuk-body">@messages("agent.sign-up-complete.whatNow.para2") </p>
 
         @form(action = postAction) {
-            @buttonHelper(messages("agent.base.sign_up_another_client"))
+            @buttonHelper(None, messages("agent.base.sign_up_another_client"))
```

Comment:
```suggestion
            @buttonHelper(None, messages("agent.base.sign_up_another_client"))
```
Instead of specifying None in each of these cases, could you remove the None and say buttonText = messages("...") instead?

Reply:
done